### PR TITLE
ENH: Add new PCASpy server for running the tutorial.

### PR DESCRIPTION
**Pull request by nstelter-slac**
Server mimics simulated motor and camera PV's.
Also brings over tutorial files to this main repo.

This allows us to drop the existing tutorial VM.

to run tutorial:
_export EPICS_CA_MAX_ARRAY_BYTES=10000000
./examples/testing_ioc/pydm-tutorial-ioc 
in another terminal...
export EPICS_CA_MAX_ARRAY_BYTES=10000000
pydm examples/tutorial/main.ui_

### CODE REVIEW

**Comment by nstelter-slac** [comment]
still need todo:

-rebase on documentation update (https://github.com/slaclab/pydm/pull/1019) and update tutorial documentation
-add units to expert_motor.ui display

File: docs/source/widgets/PyDMWidget.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ API Documentation
    :show-inheritance:
 
 PyDMToolTip Instructions
-=================
+=================Z
```
Comment (jbellister-slac) [inline docs/source/widgets/PyDMWidget.rst:13]:
```suggestion
=================
```

File: docs/source/widgets/archiver_timeplot.rst [hunk 2]
```diff
@@ -46,7 +46,7 @@ be plotted as bars to show the full range of data represented by each point. As
 of 365, a request for a year of data for a PV that updates every second would return roughly
 365 points each of which will contain the min and max of that day's data to plot the full range represented.
 
-.. figure:: /_static/widgets/archiver_time_plot/archiver_time_plot.gif
+.. figure:: /_static/widgets/archiver_time_plot/archiver_plot.gif
```
Comment (jbellister-slac) [inline docs/source/widgets/archiver_timeplot.rst:49]: Thank you for fixing this!

File: docs/source/widgets/index.rst [hunk 3]
```diff
@@ -17,6 +17,8 @@ Display Widgets
    related_display_button.rst
    scale.rst
    symbol.rst
+   analog_indicator.rst
+   nt_table.rst
```
Comment (jbellister-slac) [inline docs/source/widgets/index.rst:21]: Thanks for adding these

**Review by jbellister-slac — COMMENTED** [review]
Looks great, thanks for this! Stepped through the guide and everything is still working well, just some very minor things below

**Comment by nstelter-slac** [comment]
applied suggestions, thanks for the review!

**Pull request merged**